lib/resources/example_json/r4_examples_appointment.rb

```diff
@@ -321,6 +321,29 @@ module Resources
       }
     ]
 
+    R4_APPOINTMENT_VIDEO_VISIT_REPLACE_PATCH ||= [
+        {
+          "op": "replace",
+          "path": "/contained/0/telecom/0/value",
+          "value": "http://providerlink.vmr.net"
+        },
+        {
+            "op": "replace",
+            "path": "/contained/1/telecom/0/value",
+            "value": "http://patientlink.vmr.net"
+        },
+        {
+            "op": "replace",
+            "path": "/contained/0/telecom/0/period/start",
+            "value": "2019-07-13T08:00:00.000Z"
+        },
+        {
+            "op": "replace",
+            "path": "/contained/0/telecom/0/period/end",
+            "value": "2019-07-13T08:10:00.000Z"
+        }
+      ]
```

Comment:
Let's get the formatting fixed here. The body of each section should be indented 2 tabs (4 spaces) and this entire section should have been indented 1 tab (2 spaces) instead of 2 tabs.

Reply:
hope this is better https://github.com/cerner/fhir.cerner.com/pull/385/commits/d5c2ac83d55721b2bfd71dd39fc313636738e90d